scarlet2/plot.py

```diff
@@ -632,18 +647,20 @@ def sources(
             assert (info["prior"] is not None), "Must use a prior to get a hallucination score"
             
             # Show the unrendered model in it's bbox
-            hallucination = hallucination_score(scene, observation, k)
+            hallucination, metric = hallucination_score(scene, observation, k)
             extent = get_extent(src.morphology.bbox)
             true_max = np.max(np.abs(hallucination))
             im = ax[k-skipped][panel].imshow(hallucination,
-                vmin=-true_max, 
-                vmax=true_max,
-                cmap="RdBu_r",
+                norm=colors.SymLogNorm(linthresh=1, linscale=1,
+                        vmin=-true_max, vmax=true_max, base=10),
+                cmap="RdBu",
                 extent=extent,
                 origin="lower",
             )
             colorbar = plt.colorbar(im, ax=ax[k-skipped][panel])
-            ax[k-skipped][panel].set_title("Hallucination Source {}".format(k), fontsize=f_size)
+            colorbar.set_label('hallucination score', rotation=270, labelpad=20, fontsize=f_size)
+            title = 'confidence: ' + str(jnp.round(metric,3))
```

Comment:
In keeping with the other labels: uppercase "Confidence", please

Reply:
Updated